ozaria/site/components/teacher-dashboard/BaseTeacherLicenses/PageLicenses.vue

```diff
@@ -80,6 +92,14 @@ export default {
       >
         {{ $t("courses.get_enrollments") }}
       </primary-button>
+      <primary-button
+        v-if="showGetTestLicense"
+        class="get-test-license"
+        @click="$emit('getTestLicense', { teacherId } )"
```

Comment:
why do we need teacherId, is it different than me.id in any case?

Reply:
i'm not sure, this `teacherId` is from vuex store and having very complicated cases so I cannot say it always to be me.id lol

Reply:
maybe school admin can see teachers' licenses without spy them ? so me.id is different?